docs/source/auto_examples/TC_Tracks_Intensity.ipynb

```diff
@@ -26,7 +26,7 @@
id=str(tc.stormid[j],encoding)\n    print(tcstormid)\n    if tcstormid=='AL' and tc.count[j]== 2 and tc.thresh[j]==34 and tc.leadtime[j]<=9000:\n        tclon=float(tc.lon[j][1:5])*0.1*(-1)\n        tclat=float(tc.lat[j][1:4])*0.1\n        lon, lat = tclon, tclat\n        xpt, ypt = m(lon, lat)\n        lonpt, latpt = m(xpt, ypt, inverse=True)\n        xs1.append(lon)\n        ys1.append(lat)  \ncs2=m.plot(xs1, ys1, linestyle='--',color='Red',label='MRW_GFSv15p2')\n\n# Read the vortex center, lat and lon, from tc-tracker results for GFSv15p2 to make colored dots along with the tracks  \nxs1 = []\nys1 = []\nxs2 = []\nys2 = []\ntclon =[]\ntclat=[]\ncount=0\nfor j in range(len(tc.ws)):\n    tcstormid=str(tc.stormid[j],encoding)\n    if tcstormid=='AL' and tc.count[j]==2 and tc.thresh[j]==34 and tc.leadtime[j]<=9000:\n        tclon=float(tc.lon[j][1:5])*0.1*(-1)\n        tclat=float(tc.lat[j][1:4])*0.1\n        lon, lat = tclon, tclat\n        xpt, ypt = m(lon, lat)\n        lonpt, latpt = m(xpt, ypt, inverse=True)\n        xs1.append(lon)\n        ys1.append(lat)\n        m.plot(xs1[count], ys1[count], marker='o',color=col[j,:])\n        count=count+1\n        print(count) \n\n# Read the vortex center, lat and lon, from tc-tracker results for GFSv16beta \nfor j in range(len(tc2.ws)):\n    tc2stormid=str(tc2.stormid[j],encoding)\n    print(tc2stormid)\n    if tc2stormid=='AL' and tc2.count[j]== 2 and tc2.thresh[j]==34 and tc2.leadtime[j]<=9000:\n        tc2lon=float(tc2.lon[j][1:5])*0.1*(-1)\n        tc2lat=float(tc2.lat[j][1:4])*0.1\n        lon2, lat2 = tc2lon, tc2lat\n        xpt2, ypt2 = m(lon2, lat2)\n        lonpt2, latpt2 = m(xpt2, ypt2, inverse=True)\n        xs12.append(lon2)\n        ys12.append(lat2)  \ncs22=m.plot(xs12, ys12, linestyle='--',color='Blue',label='MRW_GFSv16beta')\n\n# Read the vortex center, lat and lon, from tc-tracker results for GFSv16beta to make colored dots along with the tracks \nxs12 = []\nys12 = []\nxs22 = []\nys22 = []\ntc2lon =[]\ntc2lat=[]\ncount2=0\nfor j in range(len(tc2.ws)-1):\n    tc2stormid=str(tc2.stormid[j],encoding)\n    if tc2stormid=='AL' and tc2.count[j]==2 and tc2.thresh[j]==34 and tc2.leadtime[j]<=9000:\n        tc2lon=float(tc2.lon[j][1:5])*0.1*(-1)\n        tc2lat=float(tc2.lat[j][1:4])*0.1\n        lon2, lat2 = tc2lon, tc2lat\n        xpt2, ypt2 = m(lon2, lat2)\n        lonpt2, latpt2 = m(xpt2, ypt2, inverse=True)\n        xs12.append(lon2)\n        ys12.append(lat2)\n        m.plot(xs12[count2], ys12[count2], marker='o',color=col2[j,:])\n        count2=count2+1\n\n# Draw coastline\nm.drawcoastlines()\nm.drawcountries()\nm.drawstates()\nm.drawmapboundary(fill_color='#99ffff')\nm.fillcontinents(color='white',lake_color='#99ffff')\ncolors.set_array([])\n\n# Show and save the plot\nplt.legend()\nplt.title('Hurricane Barry Tracks from 00Z 11 Jul to 18Z 14 Jul 2019')\nplt.colorbar(colors,fraction=0.035,pad=0.04,label='vortex maximum 10-m wind (kt)')\nplt.show()\nplt.savefig('Tracker_Barry_ufsv1.png')"
```

Comment:
Is this change intended? Wasn't there before the BestTrack --> Best Track update.

Reply:
From: Read the vortex center, lat and lon, from BestTrack data
To: Read the vortex center, lat and lon, from Best Track data
If you scroll to the right, you can see the target phrase changed in this auto-generated ipynb script.